config_defaults_inc.php

```diff
@@ -2791,6 +2791,12 @@
 $g_roadmap_view_threshold = VIEWER;
 
 /**
+ * threshold for viewing timeline
+ * @global integer $g_view_timeline_threshold
+ */
+$g_view_timeline_threshold = VIEWER;
```

Comment:
I would rename this to timeline_view_threshold (similar to roadmap_view_threshold).